graph/src/components/store.rs

```diff
@@ -1716,3 +1743,47 @@ impl LfuCache<EntityKey, Option<Entity>> {
         }
     }
 }
+
+/// Determines which columns should be selected in a table.
+#[derive(Clone, Debug, PartialEq, Eq, Hash)]
+pub enum ColumnNames {
```

Comment:
I know this is going to be an icky change, but this should really be called `AttributeNames` or `Attributes` - we are operating at the GraphQL level here, which deals in (entity) types and attributes; it's only the relational store level that operates at the level of tables and columns.

Reply:
I agree. Leo had the same impression, too.